src/routes/docs/+page.svelte

```diff
@@ -15,46 +15,31 @@
     const tutorials: CodeCardProps[] = [
         {
             href: '/docs/tutorials/react',
-            cover: {
-                src: '/images/tutorials/react.png',
-                alt: 'react logo'
-            },
+            src: '/images/tutorials/react.png',
```

Comment:
I think cover worked best. It may be confusing to see href and src attributes in the same object